doc/install/fast-install-and-first-run.rst

```diff
@@ -32,22 +47,24 @@ Install
    $ cd ivre
    $ python setup.py build
    $ sudo python setup.py install
+   $ echo 'DB_FLOW = "neo4j://neo4j:<new password>@localhost:7474/"' | sudo tee /etc/ivre.conf
```

Comment:
This is going to be unnecessary very soon (#685).

Reply:
```suggestion
   $ echo 'DB_FLOW = "neo4j://neo4j:<new password>@localhost:7474/"' | sudo tee /etc/ivre.conf # Soon unnecessary after adding MongoDB as the flow database in coming IVRE release. 
```

Reply:
#685 has been merged, this can be removed.

Reply:
Okay